app/src/main/kotlin/com/hippo/ehviewer/ui/scene/GalleryCommentsFragment.kt

```diff
@@ -401,6 +402,7 @@ fun GalleryCommentsScreen(galleryDetail: GalleryDetail, navigator: NavController
                         onValueChange = { userComment = it },
                         modifier = Modifier.weight(1f).padding(keylineMargin),
                         textStyle = MaterialTheme.typography.bodyLarge.merge(color = color),
+                        cursorBrush = SolidColor(MaterialTheme.colorScheme.primary),
```

Comment:
Why not use `TextFieldDefaults.colors()`

Reply:
We have a primaryContainer background,
and this is not standard material3 textfield usage